webapp/src/dogma/features/auth/authSlice.ts

```diff
@@ -53,13 +58,26 @@ export const login = createAsyncThunk('/auth/login', async (params: LoginParams,
   return false;
 });
 
+export const logout = createAsyncThunk('/auth/logout', async () => {
+  const response = await axios.post('/api/v1/logout', null, {
+    validateStatus: (status) => status < 500,
+  });
+  if (response.status === HttpStatusCode.Ok) {
+    removeSessionId();
+    goToLoginPage();
+    return true;
+  }
+  alert('Problem logging out. Please try again.');
+  return false;
+});
+
 interface UserSessionResponse {
   isAuthorized: boolean;
   user?: UserDto;
 }
 
 export const validateSession = createAsyncThunk<UserSessionResponse>('/auth/validate_session', async () => {
-  const response = await axios.get('/security_enabled', {
+  const response = await axios.get('/api/security_enabled', {
```

Comment:
If we are going to `api` prefix to `security_enabled`, what do you think of adding a version prefix as well?

Reply:
https://github.com/line/centraldogma/blob/c59d2a1defdb7eda3a27959eca0630fd261034cd/server/src/main/java/com/linecorp/centraldogma/server/CentralDogma.java#L648

Reply:
sure, updated.

Reply:
Actually I have changed it back to make it work with ShiroCentralDogmaTestServer and SimpleCentralDogmaTestServer. Let's add the prefix in a future PR if we decide to do so.